### PR TITLE
Update sample data

### PR DESCRIPTION
**Pull request by fusin02**

### CODE REVIEW

**Comment by jhqiu21** [comment]
LGTM! Add new event data to show our functionality.

**Comment by Haris-Irfan** [comment]
LGTM!

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -25,33 +26,36 @@ public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                 new Address("Blk 30 Geylang Street 29, #06-40"),
-                getTagSet("friends")),
+                getTagSet("Celebrity")),
```
Comment (seantham21) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:29]: I like how you have changed the context of our sample data to fit our application!

**Review by seantham21 — COMMENTED** [review]
LGTM!

**Pull request merged**